pom.xml

```diff
@@ -53,7 +53,7 @@
         <protoc.version>3.5.1</protoc.version> <!-- Same version as grpc-proto -->
         <grpc.contrib.version>0.8.1</grpc.contrib.version>
         <rxjava.version>2.1.10</rxjava.version>
-        <reactor.version>3.1.5.RELEASE</reactor.version>
+        <reactor.version>3.2.5.RELEASE</reactor.version>
```

Comment:
⚠️ unused. please also make sure that it is understood this is a change that breaks backward compatibility (3.2.x vs 3.1.x), cc @rmichela

Reply:
(note the version variable is unused in some downstream poms)

Reply:
How is 3.2 breaking from 3.1? Not semver?

Reply:
@OlegDokuka Why was it necessary to upgrade the version of reactor-core?

Reply:
probably it is good to have but it is not necessary

Reply:
Is it perhaps needed to fix "VerifySubscriber timed out" errors? If so, why?

Reply:
I'm not sure. I'm still fighting with fused impl `AbstractSubscriberAndProducer` processor, so the newer version or reactor will not fix that

Reply:
@rmichela unfortunately we live with a legacy of Reactor 2. So Reactor 3 doesn't follow semver, but rather something along the lines of DEEP_ARCHITECTURAL_CHANGE.BREAKING_CHANGE.MAINTENANCE, although we try to keep the breaking changes minimal ([3.1](https://github.com/reactor/reactor-core/releases/tag/v3.1.0.RELEASE) was heavier in that respect than [3.2](https://github.com/reactor/reactor-core/releases/tag/v3.2.0.RELEASE)